simulator/risc_v/riscv_instr.cpp

```diff
@@ -122,99 +178,159 @@ struct RISCVTableEntry
     Src2 src2;
     Dst  dst;
     uint32 mem_size;
+    Base base = RV_ANY;
```

Comment:
Same here. `base` usually refers to something which is a base (e.g. base class).